cdk/kraken/src/deploy.ts

```diff
@@ -64,12 +78,23 @@ export class DeployJob extends CheckoutJob {
           },
         },
         {
-          name: "Deploy",
-          run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
+          name: 'Push to kube-manifests repository',
+          run: dedent`cd kube-manifests
+          mkdir -p \${{ github.repository }}
+          cp -r ../k8s/dist/ \${{ github.repository }}
 
           # get repo name from synth step
           RELEASE_NAME=\${{ steps.synth.outputs.RELEASE_NAME }}
 
+          git add \${{ github.repository }}
+          git commit -m "chore(k8s): deploy $RELEASE_NAME"
```

Comment:
Just a suggestion, but if we change the commit message to be something like "deploy pennlabs/repo-name@git-sha", we'll get a direct link to the commit that's being deployed. It might be a small QoL improvement.